fsharp-backend/src/LibBackend/EventQueue.fs

```diff
@@ -155,18 +155,32 @@ module WorkerStates =
 
 
 
-// None in case we want to log on a timer or something, not
-// just on enqueue/dequeue
+/// <summary>
+/// Logs the current status of what's enqueued for the the given queue,
+/// both at the "worker level" (ready to be picked up by workers),
+/// and at the "canvas level" (ready to be enqueued _for_ a worker).
+/// </summary>
+///
+/// <remarks>
+/// `host` is optional because we have it when we dequeue, but not enqueue.
+/// </remarks>
+///
+/// WHATISTHIS why do we have both 'name' and 'host'?
+/// is it anticipation of a queue existing in one canvas, but being used in another?
```

Comment:
Pending question

Reply:
I think `host` should be called `canvasName`, but `name` is the name of the worker (as in `emit_v1 "workerName"`).